types/react-big-calendar/index.d.ts

```diff
@@ -536,24 +536,19 @@ export class Calendar<TEvent extends object = Event, TResource extends object =
     CalendarProps<TEvent, TResource>
 > {}
 
-export interface components {
-    timeSlotWrapper: React.ComponentType;
-    dateCellWrapper: React.ComponentType;
-    eventWrapper: React.ComponentType<Event>;
-}
```

Comment:
not sure this should be removed, this is an interface for the object library is exporting

https://github.com/jquense/react-big-calendar/blob/537c6f3f08344191588f2ab3ad52667de7ee261f/src/index.js#L4-L8

Reply:
Good point. I did not see that definition 👍

Reply:
@michalak111 , I have reverted this change.